Website/src/rendering/src/components/Checkout/LineItemCard.tsx

```diff
@@ -0,0 +1,164 @@
+    const lineItem = props.lineItem;
+    if (!lineItem) {
+      return null;
+    }
+    if (lineItem.Variant?.xp?.Images?.length) {
+      return lineItem.Variant.xp.Images[0].Url;
+    }
+    if (lineItem.Product?.xp?.Images?.length) {
+      return lineItem.Product.xp.Images[0].Url;
+    }
+    return null;
+  };
+
+  const handleRemoveLineItem = useCallback(async () => {
+    setLoading(true);
+    await dispatch(removeLineItem(props.lineItem.ID));
+  }, [dispatch, props.lineItem]);
+
+  const handleUpdateQuantity = useCallback(
+    async (quantity: number) => {
+      setLoading(true);
+      await dispatch(
+        patchLineItem({
+          lineItemID: props.lineItem.ID,
+          partialLineItem: { Quantity: quantity },
+        })
+      );
+      setLoading(false);
+    },
+    [dispatch, props.lineItem]
+  );
+
+  // TODO: add branded placeholder img
+  const productImage = (
+    <img
+      src={getImageUrl() || 'https://via.placeholder.com/100'}
+      alt={props.lineItem.Product.Name}
+    ></img>
+  );
+
+  const quantityInput = props.editable && product && (
+    <QuantityInput
+      controlId={`${props.lineItem.ID}_quantity`}
+      quantity={props.lineItem.Quantity}
+      disabled={loading}
+      onChange={handleUpdateQuantity}
+      priceSchedule={product.PriceSchedule}
+    />
+  );
+
+  const btnRemove = (
+    <button
+      className="btn-remove"
+      aria-label="Remove Line Item"
+      type="button"
+      disabled={loading}
+      onClick={handleRemoveLineItem}
+    >
+      <FontAwesomeIcon icon={faTrashAlt} />
+    </button>
+  );
+
+  // TODO: add functionality to button
+  const btnWishList = (
+    <button className="btn-wishlist" aria-label="Add to Wish List" type="button">
+      <FontAwesomeIcon icon={faHeart} />
+    </button>
+  );
+
+  // TODO: add functionality to button
+  const btnSaveLater = (
+    <button className="btn-later" aria-label="Save for Later" type="button">
+      <FontAwesomeIcon icon={faHistory} />
+    </button>
+  );
+
+  const quantityBlock = props.editable ? (
+    <>
+      {quantityInput}
+      {btnRemove}
+      {btnWishList}
+      {btnSaveLater}
+    </>
+  ) : (
+    <p className="quantity-static">
+      <span className="quantity-label">Quantity: </span>
+      <span className="quantity-num">{props.lineItem.Quantity}</span>
+    </p>
+  );
+
+  const giftCheckbox = props.editable && (
+    <GiftCheckboxLineItem lineItem={props.lineItem}></GiftCheckboxLineItem>
+  );
+
+  // TODO: add functionality to input
+  const userComment = (
+    <input type="text" placeholder="Text input for user..." className="user-comment" />
+  );
+
+  // TODO: add functionality to field
+  const quantityAlert = <p className="quantity-alert">Only 3 left!</p>;
+
+  // TODO: add functionality to block (specs to return price and discount)
+  const priceBlock = (
+    <div className="price">
+      <span className="price-base">$299.99</span>
+      <span className="price-discount">-20%</span>
```

Comment:
I know this request is not part of the design. Would it please be possible to add some spacing between the base price and the discount?